mne/channels/montage.py

```diff
@@ -348,6 +347,40 @@ def get_positions(self):
         )
         return positions
 
+    def add_estimated_fiducials(self, subject, subjects_dir):
+        """Estimate fiducials based on FreeSurfer ``fsaverage`` subject.
+
+        Parameters
+        ----------
+        %(subject)s
+        %(subjects_dir)s
+
+        Notes
+        -----
+        Since MNE uses the FIF data structure, it relies on the ``head``
+        coordinate frame. Any coordinate frame can be transformed
+        to ``head`` if the fiducials (i.e. LPA, RPA and Nasion) are
+        defined. One can use this function to estimate those fiducials
+        and then use ``montage.get_native_head_t()`` to get the
+        head <-> MRI transform.
+        """
+        from mne.coreg import get_mni_fiducials
+
+        # get coordframe and fiducial coordinates
+        montage_bunch = _get_data_as_dict_from_dig(self.dig)
+
+        # estimate LPA, nasion, RPA from FreeSurfer fsaverage
+        fids_mri = get_mni_fiducials(subject, subjects_dir)
+
+        # set the coordinate frame based on the coordinate frame
+        # XXX: is this step right? It for example would set mri -> mni_tal
+        for fid in fids_mri:
+            fid['coord_frame'] = montage_bunch.coord_frame
+        _validate_type(item=fids_mri, types=list, item_name='dig')
+
+        # add those digpoints to montage
+        self.dig.extend(fids_mri)
```

Comment:
Might be better to insert these at the beginning, I think that's usually where we put fiducials in the dig list.